### ISSUE TITLE
Dim selection error (maybe) when distilling

### ISSUE THREAD

**Issue by jinyu121** [kshmelkov/incremental_detectors#3]
Let's see https://github.com/kshmelkov/incremental_detectors/blob/master/network.py#L59-L87

Say `self.num_classes = 15`, and  `self.subnet.num_classes = 10`, so we use `logits = self.logits_for_distillation[:, :cached_classes+1]` for distillation. 

But wait, `[:, :cached_classes+1]`? It is `0-6` now,  rather than `0-11`. Should we use `0-11` here?

```
def compute_distillation_crossentropy_loss(self):
        cached_classes = self.subnet.num_classes
        ...

def compute_distillation_bbox_loss(self):
        cached_classes = self.subnet.num_classes
        ...
```

**Comment by pyun-ram** [comment]
I agree with youi @jinyu121

**Comment by resona9970** [comment]
thank you, i think you are right !
